src/pages/Home.jsx

```diff
@@ -1,91 +1,141 @@
 // eslint-disable-next-line import/no-extraneous-dependencies
 import faker from 'faker'
 import React from 'react'
+import { useState } from 'react'
+import { useQueryClient, useQuery } from 'react-query'
+import axios from 'axios'
+import classNames from 'classnames'
 
 function Home() {
+  const PAGE_LIMIT = 5
+  const [offset, setOffset] = useState(0)
+  const [selectedTag, setSelectedTag] = useState(null)
+  const queryClient = useQueryClient()
+  let params = {
+    offset: offset * PAGE_LIMIT,
+    limit: PAGE_LIMIT
+  }
+
+  if (selectedTag) {
+    params = Object.assign(params, { tag: selectedTag })
+  }
+  if (selectedTag) {
+    Object.assign(params, {})
+  }
+  const articlesQuery = useQuery(`get-articles-list-${offset}-${selectedTag}`, () => axios.get('https://conduit.productionready.io/api/articles', {
+    params
+  }), {
+    placeholderData: {
+      data: {
+        articles: [],
+        articlesCount: 0
+      }
+    }
+  })
```

Comment:
```suggestion
  const articlesQuery = useQuery(['get-articles-list', { offset, selectedTag }], () => axios.get('https://conduit.productionready.io/api/articles', {
    params
  }), {
    placeholderData: {
      data: {
        articles: [],
        articlesCount: 0
      }
    }
  })
```

Reply:
Lol, truly more elegant , your suggestion, didn't know it was possible, hate the string based caching key